core_ui/src/main/java/com/trempel/core_ui/exceptions/ServiceExceptionDialog.kt

```diff
@@ -0,0 +1,32 @@
+package com.trempel.core_ui.exceptions
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentResultListener
+import androidx.lifecycle.LifecycleOwner
+import com.example.core_ui.R
+import com.example.core_ui.databinding.ServiceExceptionDialogBinding
+
+class ServiceExceptionDialog : DialogFragment() {
```

Comment:
Can we have single error dialog but simply hide retryButton if we don't need retry button?